docs/layers/identity/deploy.mdx

```diff
@@ -62,28 +62,34 @@ These components are expected to be used together to provide fine-grained role d
     ```
 
     This will generate AWS configuration to the `rootfs` directory locally.
```

Comment:
Do we explain anywhere why we generate these configs?

Reply:
I dont think we do. I'll add a bit of context to this line